servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H1ProtocolConfigBuilder.java

```diff
@@ -130,14 +131,39 @@ public H1ProtocolConfigBuilder trailersEncodedSizeEstimate(final int trailersEnc
         return this;
     }
 
+    /**
+     * Defines if an HTTP/1.1 response with <a href="https://tools.ietf.org/html/rfc7230#section-6.1">
+     * Connection: close</a> and <a href="https://tools.ietf.org/html/rfc7230#section-3.3.1">
```

Comment:
Is it necessary to special case this only when `Connection: close` header is present?

Reply:
We are adding an exception to spec rules, it seems we can make this more generic than to follow the spec.

Reply:
The question: is it worth making it more generic before we know the use-case? I would prefer to keep the scope of each exception minimal as broader scope of not following RFC may introduce undesired behavior.

Reply:
Discussed offline and decided to remove requirement for the presence of `Connection: close` header.
Fixed in https://github.com/apple/servicetalk/pull/1060/commits/64fb4cc1ab4a8c3316a094560bd531dd236615f3